omod/src/test/java/org/openmrs/module/fhir2/providers/r3/ConditionFhirR3ResourceProviderWebTest.java

```diff
@@ -229,6 +240,67 @@ public void shouldCreateNewConditionGivenValidConditionResource() throws Excepti
 		assertThat(response.getStatus(), is(201));
 	}
 	
+	@Test
+	public void updateCondition_shouldUpdateExistingCondition() throws Exception {
+		String conditionJson;
+		try (InputStream is = this.getClass().getClassLoader().getResourceAsStream(JSON_UPDATE_CONDITION_PATH)) {
+			Objects.requireNonNull(is);
+			conditionJson = IOUtils.toString(is, StandardCharsets.UTF_8);
+		}
+		when(conditionService.update(conditionJson, any(org.hl7.fhir.r4.model.Condition.class))).thenReturn(condition);
```

Comment:
As the error indicates, you can't combine an `ArgumentMatcher` (e.g., the `any()`) with a plain value. Also, in this case, `conditionService.update()` will be called with the UUID passed into the request, rather than the full JSON of the request.

```suggestion
		when(conditionService.update(eq(CONDITION_UUID), any(org.hl7.fhir.r4.model.Condition.class))).thenReturn(condition);
```